ceph.spec.in

```diff
@@ -704,12 +705,14 @@ Recommends:     python%{python3_pkgversion}-saml
 %if 0%{?fedora} || 0%{?rhel} <= 8
 Recommends:     python%{python3_pkgversion}-grpcio
 Recommends:     python%{python3_pkgversion}-grpcio-tools
+Requires:       python%{python3_pkgversion}-xmltodict
```

Comment:
Just noticed this. Why are we adding it to 3 different places? I think the first one should have been enough isnt it

Reply:
I wasn't sure to be honest, so just added it for all versions
Should I leave it only for `%if 0%{?fedora} || 0%{?rhel} >= 9`?

Reply:
You can keep the one that is below the jmespath.

Reply:
done